bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/utils/SvnUtil.java

```diff
@@ -328,6 +269,99 @@ private static boolean setupWorkingCopy(SVNMergeUnit mergeUnit, ISvnClient clien
 		return LogUtil.exiting(cancel);
 	}
 
+	private static List<Path> getPaths(List<String> targetFile, File workingCopy) {
+		List<Path> paths = new ArrayList<>(targetFile.size() + 1);
+		if (!targetFile.isEmpty()) {
+			// update neededFiles AND working copy folder (so we don't end up
+			// with a mixed
+			// revision working copy)
+			paths.add(workingCopy.toPath());
+
+			for (String target : targetFile) {
+				final List<Path> pathsToAdd = new ArrayList<>();
+				Path path = Paths.get(target);
+				while (path != null) {
+					pathsToAdd.add(Paths.get(workingCopy.getPath(), path.toString()));
+					path = path.getParent();
+				}
+				Collections.reverse(pathsToAdd);
+				paths.addAll(pathsToAdd);
+			}
+		} else {
+			LOGGER.fine("mergeUnit has no needed files for the working copy."); //$NON-NLS-1$
+		}
+		return paths;
+	}
+
+	private static List<Path> getMissingFiles(List<Path> paths, SVNMergeUnit mergeUnit, File workingCopy) {
+		List<Path> missingFiles = new ArrayList<>();
+		if (paths != null) {
+			for (Path path : paths) {
+				final String relPath = Paths.get(workingCopy.toString()).relativize(path).toString().replace('\\', '/');
+				final boolean isAddedPath = mergeUnit.getTargetFilesToAdd().contains('/' + relPath + '/')
+						|| mergeUnit.getTargetFilesToAdd().contains('/' + relPath);
+				if (!path.toFile().exists() && !isAddedPath) {
```

Comment:
Basic Idea behind this change was to avoid filling up of stack with errors when the file does not exist.
Not sure how this is handled in Java 11 though to be honest. Can change if you think this is unnecessary.